go/coinstacks/osmosis/api/handlers.go

```diff
@@ -155,7 +172,7 @@ func (h *Handler) GetTxHistory(pubkey string, cursor string, pageSize int) (api.
 			GasUsed:       t.TendermintTx.TxResult.GasUsed,
 			Index:         int(t.TendermintTx.GetIndex()),
 			Memo:          t.SigningTx.GetMemo(),
-			Messages:      osmosis.Messages(t.CosmosTx.GetMsgs()),
+			Messages:      cosmos.Messages(t.CosmosTx.GetMsgs()),
```

Comment:
Is this named `cosmos` because we're referring to Cosmos SDK vs. osmosis the chain?

Reply:
Yes, this and the Messages call in `handleTx` should be using `osmosis.Messages`